collections/genes.coffee

```diff
@@ -13,3 +13,12 @@ module.exports = class Genes extends Backbone.Collection
   model: Gene
 
   url: "#{sd.API_URL}/api/v1/genes"
+
+  groupByFamily: ->
```

Comment:
👌 always appreciate moving logic to the model—especially data-oriented like this.